### PR TITLE
Fix Floating Point Precision Errors Caused by Unit Systems on Output:Table:SummaryReports AirLoopComponentLoadSummary Table

### LINKED ISSUE #6475: Engineering Checks in Component Load Summary Reports need more precision
@JasonGlazer 

Issue overview
--------------
'Airflow per total capacity' values often report to 0.0001 or 0.000 for zones. This is not enough precision and gives bad values when converting to IP. 

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by mitchute**
PR fixes #6475. Fixed floating point precision on Output:Table:SummaryReports AirLoopComponentLoadSummary has problems due to unit conversions.

See the following examples for the same simulation when using SI and I-P units.

Screen Shot 2020-02-05 at 2 35 59 PM

Screen Shot 2020-02-05 at 2 37 41 PM

This PR uses the shiny new ```fmtlib::fmt``` third-party library to convert some of the values to scientific notation. See the following for the proposed changes.

Screen Shot 2020-02-05 at 2 44 48 PM

Screen Shot 2020-02-05 at 2 44 06 PM

Also, note the ```Outside Air (%)``` field was changed to ```Outside Air Fraction (-)``` since this value is actually a fractional value.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by mitchute** [comment]
@JasonGlazer could you take a look at this commit (e50e80537c808475d942ba33459a4f0bf4857ad6) sometime? I thought this would at least get me close to a working unit test, but it's returning a 0 sized vector and I'm at the limits of my SQL knowledge.

**Comment by JasonGlazer** [comment]
Sorry, I'm no SQL expert. I usually copy and modify some other SQL that is working and cross my fingers :-)

**Comment by mitchute** [comment]
@JasonGlazer Thanks for taking a look!

**Comment by mitchute** [comment]
Ready for review.

File: src/EnergyPlus/OutputReportTabular.cc [hunk 1]
```diff
@@ -62,6 +62,9 @@
 #include <ObjexxFCL/string.functions.hh>
 #include <ObjexxFCL/time.hh>
 
+// Third-party Headers
+#include <fmt/format.h>
+
```
Comment (Myoldmopar) [inline src/EnergyPlus/OutputReportTabular.cc:67]: :+1:

File: src/EnergyPlus/OutputReportTabular.cc [hunk 2]
```diff
@@ -14119,36 +14122,27 @@ namespace OutputReportTabular {
...
                     rowHead(2) = "Airflow per Floor Area [m3/s-m2]";
                     rowHead(3) = "Airflow per Total Capacity [m3/s-W]";
                     rowHead(4) = "Floor Area per Total Capacity [m2/W]";
                     rowHead(5) = "Total Capacity per Floor Area [W/m2]";
-                    //					rowHead( 6 ) = "Chiller Pump Power per Flow [W-s/m3]"; // facility only
-                    //					rowHead( 7 ) = "Condenser Pump Power per Flor [W-s/m3]"; // facility only
                     rowHead(6) = "Number of People";
                 } else {
-                    rowHead(1) = "Outside Air (%)";
+                    rowHead(1) = "Outside Air Fraction [fraction]";
```
Comment (Myoldmopar) [inline src/EnergyPlus/OutputReportTabular.cc:14132]: Good change here

File: tst/EnergyPlus/unit/OutputReportTabular.unit.cc [hunk 3]
```diff
@@ -7131,7 +7131,13 @@ TEST_F(SQLiteFixture, OutputReportTabular_WriteLoadComponentSummaryTables_AirLoo
         EXPECT_EQ(std::get<2>(v), oa_db) << "Failed for TableName=" << tableName << "; ReportName=" << reportName;
     }
 
+    // https://github.com/NREL/EnergyPlus/pull/7741
+    std::string query_2("SELECT Value From TabularDataWithStrings"
+                        "  WHERE TableName = 'Engineering Checks for Cooling'"
+                        "  AND RowName = 'Outside Air Fraction';");
 
+    auto result = queryResult(query_2, "TabularDataWithStrings")[0][0];
+    EXPECT_EQ(result, "0.0000");
```
Comment (Myoldmopar) [inline tst/EnergyPlus/unit/OutputReportTabular.unit.cc:7140]: Glad this is working!

**Review by Myoldmopar — APPROVED** [review]

**Comment by Myoldmopar** [comment]
This just gets tablediffs locally on my machine as well.  Merging this in.

**Pull request merged**